web/modules/custom/dpl_react_apps/dpl_react_apps.module

```diff
@@ -11,6 +11,7 @@
 
 use Drupal\Core\GeneratedUrl;
 use Drupal\Core\Url;
+use Drupal\dpl_fees\DplFeesSettings;
```

Comment:
Since you are introducing dpl_fees settings in the dpl_react_apps module you need to add the dpl_fees module as a dependency [here](https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/a6de7e5518445fae08658aba3bc41a915a8d3220/web/modules/custom/dpl_react_apps/dpl_react_apps.info.yml#L11).